modules/ROOT/pages/servers-create.adoc

```diff
@@ -3,19 +3,22 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes how to use the `amc_setup` script to create a server by configuring the Runtime Manager Agent to communicate with Runtime Manager. The server communicates with Runtime Manager using the Runtime Manager Agent. To make a server visible in the Runtime Manager console and be able to manage it, you must first register it with the Runtime Manager Agent.
+You can use the `amc_setup` script to create a server by configuring the Runtime Manager Agent to communicate with Runtime Manager. The server communicates with Runtime Manager using the Runtime Manager Agent. To make a server visible in the Runtime Manager console so you can manage it, you must first register it with the Runtime Manager Agent.
 
 == Prerequisites
 
 To manage servers in a customer-hosted environment, you must perform the following:
 
 * Install Mule runtime, version 3.6 or later or API Gateway runtime, version 2.0 or later.
 * Install Runtime Manager Agent, version 1.1.1 or later.
-* (Optional) Configure any whitelisted IPs or ports required to connect to the Anypoint Platform through a firewall.
+* (Optional) Configure any whitelisted IPs or ports required to connect to Anypoint Platform through a firewall.
 
-If the server you want to add is already registered with a different instance of Runtime Manager, you cannot add it until you remove if from the other instance. To do this, delete the server from the Server tab in Runtime Manager, then delete the 'mule-agent.yaml' configuration file found on your '{RUNTIME PARENT FOLDER}/conf' folder.
+If the server you want to add is already registered with a different instance of Runtime Manager, you cannot add it until you remove if from the other instance. To do this:
 
-If your environment requires all outbound calls to go through a proxy you will need to modify the proxy settings in the 'wrapper.conf' file in the '{RUNTIME PARENT FOLDER}/conf' directory.
+. In Runtime Manager, delete the server from the *Server* tab.
+. Delete the 'mule-agent.yaml' configuration file, which can be found in your '{RUNTIME PARENT FOLDER}/conf' folder.
```

Comment:
I think the tick marks should be `mule-agent.yaml` not straight single quotes as in the original (throughout this topic).